### PR TITLE
Modify CI to skip package build for linux and outerloop builds

### PR DESCRIPTION
**Pull request by iamjasonp**

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -38,7 +38,7 @@ def project = 'dotnet/wcf'
             // so we set it first, and then call build.sh
             newCommitJob = job(Utilities.getFullJobName(project, jobName, false)) {
                 steps {
-                    shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:OSGroup=${os} /p:Configuration=${os}_${configuration}")
+                    shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:ShouldCreatePackage=false /p:OSGroup=${os} /p:Configuration=${os}_${configuration}")
```
Comment (roncain) [inline netci.groovy]: For some reason, the only command that works for me seems to be:
```
  ./build.sh /p:ShouldGenerateNuSpec=false /p:ShouldCreatePackage=false
```
I tried just ShouldCreatePackage and/or SkipBuildPackages but still got packaging errors.
If your version works for you in CI, don't block on this observation.

Comment (iamjasonp) [inline netci.groovy]: Need to test in CI; it was offline for much of my working day.

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

File: netci.groovy (hunk 1 above)
Comment (iamjasonp) [inline netci.groovy]: Incidentally, I was trying the /p:ShouldGenerateNuSpec parameter based on a hunch looking at the targets files, but didn't get a chance to try that change in CI.

Comment (iamjasonp) [inline netci.groovy]: Alrighty - confirmed that I need both  /p:ShouldGenerateNuSpec=false /p:ShouldCreatePackage=false based on running the various permutations in CI.

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by roncain** [comment]
LGTM

**Pull request merged**